packages/components/src/components/toggle-switch/toggle-switch.jsx

```diff
@@ -1,21 +1,23 @@
-import React from 'react';
+import * as React from 'react';
```

Comment:
?

Reply:
https://epicreact.dev/importing-react-through-the-ages/